Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -59,6 +59,7 @@ def runSelected(opt):
                      135.4, #Run 2 Zee ttbar
                      10021.0, #2017 tenmu
                      10024.0, #2017 ttbar
+                     10424.0, #2017 NewPix ttbar
```

Comment:
IMO, frequent renumbering is not nice, but need to happen sometimes.

Note that once 2023 begin to grow they will soon reach the 25XXX phase space of 25ns wflows in 2016.
Also, some of the other run2 ranges may need to be readjusted (we have high-stats, fast-sim etc, some are 5-digit, some are 6-digit). 

Some purge and update for 2017 will have to be done fairly soon (~Oct-Nov?) to expand to a more covering set, similar to what we have for 2016.